src/clitimer/Program.cs

```diff
@@ -0,0 +1,346 @@
+﻿// This file is part of the SoulSplitter distribution (https://github.com/FrankvdStam/SoulSplitter).
+// Copyright (c) 2022 Frank van der Stam.
+// https://github.com/FrankvdStam/SoulSplitter/blob/main/LICENSE
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Threading;
+using SoulMemory.DarkSouls1;
+using SoulMemory.DarkSouls2;
+using SoulMemory.DarkSouls3;
+using SoulMemory.Sekiro;
+using SoulMemory.EldenRing;
+
+#pragma warning disable CS0162
+
+namespace clitimer
+{
+    internal class Program
+    {
+        [STAThread]

```

Comment:
[STAThread] is not required here, I guess it came from the other cli application. It exists there to allow me to programmatically setup win-forms for testing, and winforms require some thread specific stuff.